app/src/main/res/layout/activity_bill.xml

```diff
@@ -255,32 +268,69 @@
                             android:textSize="@dimen/sp_10"
                             android:textStyle="bold"/>
 
-                        <ImageView
+                        <android.support.design.widget.TextInputLayout
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
-                            android:layout_marginTop="@dimen/dp_2"
-                            android:onClick="@{()-> viewModel.onAddBillClick()}"
-                            app:srcCompat="@drawable/ic_cart_60dp"
-                            tools:ignore="ContentDescription"/>
+                            bind:errorText="@{viewModel.formError}">
+
+                            <android.support.v7.widget.CardView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:paddingRight="@dimen/dp_10"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingRight` you should probably also define `paddingLeft` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.